components/feature/loginsexceptions/schemas/mozilla.components.feature.loginsexceptions.db.LoginExceptionDatabase/1.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
q - am i supposed to include this schema file?

Reply:
Yes, this is great, those can later be used to create older versions of the database to test migrations 👍